src/styles/stars.css

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  line-height: 14px;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin: 0 2px;
```

Comment:
It would be great to change to just margin-left: 4px not to reset left margin later for an card__element

Reply:
Yeah I remember you had already mentioned this in the class, I will correct it